### DISCUSSION TITLE
OpenClash 无法安装

### DISCUSSION THREAD

**Discussion by dmanbu** [vernesong/OpenClash#2676] [Q&A]
我用的openwrt是这个版本: https://github.com/coolsnowwolf/lede/releases/download/20200408/openwrt-R20.4.8-x64-combined-squashfs.1.rar

依赖包已经装了，但是在安装 luci-app-openclash_0.45.51-beta_all.ipk 包时，提示以下错误：
```
Collected errors:
 * pkg_init_from_file: Malformed package file /tmp/luci-app-openclash_0.45.51-beta_all.ipk.
```

请问是不支持这个版本的 openwrt 么？我该用什么版本？

**Comment by dmanbu** [comment]
已解决
我之前是用命令行安装ipk包的，后来我通过web页面上传ipk包进行安装就成功了

**Reply by Kethers** [reply]
我是18.06-5.4-SNAPSHOT版本的openwrt，想装 luci-app-openclash_0.46.079_all.ipk，也是提示这个错误，用命令行和文件系统上传安装都是同样的报错（查日志）。怀疑是openwrt版本太老了不支持这么新的openclash
